microprofile/5.0/openapi/3.0/22.0.0.1-beta-TCKResults.adoc

```diff
@@ -0,0 +1,37 @@
+:page-layout: certification
+= TCK Results
+
+As required by the https://www.eclipse.org/legal/tck.php[Eclipse Foundation Technology Compatibility Kit License], following is a summary of the TCK results for releases of MicroProfile Interoperable JWT RBAC 2.0.
+
+== Open Liberty 22.0.0.1-beta - MicroProfile Interoperable JWT RBAC 2.0 Certification Summary
+
+* Product Name, Version and download URL (if applicable):
++
+https://repo1.maven.org/maven2/io/openliberty/openliberty-runtime/22.0.0.1-beta/openliberty-runtime-22.0.0.1-beta.zip[Open Liberty 22.0.0.1-beta]
+* Specification Name, Version and download URL:
++
+link:https://download.eclipse.org/microprofile/microprofile-open-api-2.0/microprofile-openapi-spec-2.0.html[MicroProfile Open API 2.0]
```

Comment:
I think this should be 3.0-RC5 but I can only find a link for RC4? Need to ask @Emily-Jiang about this one
https://download.eclipse.org/microprofile/microprofile-open-api-3.0-RC4/microprofile-openapi-spec-3.0-RC4.html
```suggestion
link:https://download.eclipse.org/microprofile/microprofile-open-api-3.0-RC5/microprofile-openapi-spec-3.0-RC5.html[MicroProfile Open API 3.0-RC5]
```

Reply:
https://mvnrepository.com/artifact/org.eclipse.microprofile.openapi/microprofile-openapi-api/3.0-RC5

Reply:
@Emily-Jiang that is a link to an index page for the API. What we need is a link to the spec (in html). If you look here - https://download.eclipse.org/microprofile/ - there is no `microprofile-open-api-3.0-RC5`

Reply:
It should be there now